src/Tribe/Aggregator/Record/List_Table.php

```diff
@@ -513,13 +513,25 @@ public function column_source( $post ) {
 		}
 
 		$html[] = '<p>' . esc_html_x( 'via ', 'record via origin', 'the-events-calendar' ) . '<strong>' . $source_info['via']  . '</strong></p>';
+
+		$html[] = '<div class="tribe-view-links-container" style="display:flex; flex-direction:row">';
+        if ( 'scheduled' === $this->tab->get_slug() ) {
```

Comment:
:no_entry_sign: Tabs must be used to indent lines; spaces are not allowed

<sub>hash: cf98cad37340290e8481aadd8548a695</sub>